AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -140,8 +140,14 @@ extension ConfirmCreatingSessionView {
             }
         }
     }
-    
+
     func getAndSaveStartingLocation() {
+        #if targetEnvironment(simulator)
+        let krakowLat = 50.049683
+        let krakowLong = 19.944544
+        sessionContext.saveCurrentLocation(lat: krakowLat, log: krakowLong)
+        return
+        #endif
```

Comment:
I think this code is copy/pasted from develop? It will conflict, but no biggie

Reply:
I merged develop cause of conflicts